src/Samples/Sample.AspNetCore.SystemTests/Test/Base/TestBase.cs

```diff
@@ -1,92 +1,72 @@
-                        WithFileName(screenshotInfo => $"{AtataContext.Current.TestName} - {screenshotInfo.PageObjectFullName}").
-                UseTestName(() => $"[{_driverAlias}]{TestContext.CurrentContext.Test.Name}");
-#endif
-        }
-
-
-        [SetUp]
-        public void SetUp()
-        {
-#if DEBUG
-            AtataContext.Configure()
-                .UseDriver(_driverAlias)
-                    .UseBaseUrl("https://localhost:44344/")
-            .Build();
-            AtataContext.Current.Driver.Maximize();
-#elif DEV
-            AtataContext.Configure()
-                .UseDriver(_driverAlias)
-                    .UseBaseUrl("https://YourBaseUrl.com/")
-            .Build();
-            AtataContext.Current.Driver.Maximize();
-#elif RELEASE
-
-            AtataContext.Configure()
-                .UseChrome()
-                .WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Chrome) as ChromeOptions)
-                .UseBaseUrl(Environment.GetEnvironmentVariable("Swedbank.Pay.Sdk.SampleWebsite.BaseUrl", EnvironmentVariableTarget.User))
-                .Build();
-            AtataContext.Current.Driver.Maximize();
-#endif
-        }
-
-        protected TestBase(string driverAlias) => this._driverAlias = driverAlias;
-
-        [TearDown]
-        public void TearDown()
-        {
-            AtataContext.Current?.CleanUp();
-        }
-
-
-        [OneTimeTearDown]
-        public void GlobalDown()
-        {
-            foreach (Driver driverType in Enum.GetValues(typeof(Driver)))
-                WebDriverCleanerService.KillWebDriverProcess(WebDriverCleanerService.DriverNames[driverType]);
-        }
-    }
+﻿using System;
+using Atata;
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.IE;
+
+using Sample.AspNetCore.SystemTests.Services;
+
+namespace Sample.AspNetCore.SystemTests.Test.Base
+{
+    using static Drivers;
+
+    [TestFixture(DriverAliases.Firefox)]
+    [Parallelizable(ParallelScope.All)]
+    public abstract class TestBase
+    {
+        private readonly string _driverAlias;
+
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            AtataContext.GlobalConfiguration.
+                UseChrome().
+                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Chrome) as ChromeOptions).
+                UseFirefox().
+                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Firefox) as FirefoxOptions).
+                UseInternetExplorer().
+                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.InternetExplorer) as InternetExplorerOptions).
+                AddNUnitTestContextLogging().
+                    WithMinLevel(LogLevel.Info);
+        }
+
+
+        [SetUp]
+        public void SetUp()
+        {
+            //var chromeOptions = DriverOptionsFactory.GetDriverOptions(Driver.Chrome) as ChromeOptions;

```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)